### ISSUE TITLE
Could DD_VERSION have a default?

### ISSUE THREAD

**Issue by croaky** [DataDog/heroku-buildpack-datadog#198]
I'm considering setting up Datadog version tagging on my Heroku project.

I could do something like this in our deploy script:

```
# Deploy
heroku pipelines:promote --app app-staging

RELEASE=heroku config:get HEROKU_RELEASE_VERSION --app app-production
SHA=heroku config:get HEROKU_SLUG_COMMIT --app app-production | cut -c1-9
VERSION="$RELEASE-$SHA"

# Datadog
heroku config:set DD_VERSION="$VERSION" --app app-production
```

This has a downside, though: it will cycle all the dynos a second time, potentially slowing down the user experience and adding noise to our alerting

Could the buildpack set a default DD_VERSION env var if I haven't provided one? The two relevant env vars that https://devcenter.heroku.com/articles/dyno-metadata sets are:

```
HEROKU_RELEASE_VERSION:               v123
HEROKU_SLUG_COMMIT:                   abc123
```

Maybe a concatenation of those would be ideal? v123-abc123 (maybe first 9 characters of the SHA?) Would provide both the Git commit information I need to compare commits but also the Heroku release version to compare config changes.

**Comment by pcreux** [comment]
Hey @croaky ! I'm actually looking at this today. I'm adding it to `datadog/prerun.sh`:

```bash
if [ -n "$HEROKU_SLUG_COMMIT" ]; then
  DD_VERSION=$HEROKU_SLUG_COMMIT
fi
```

It seems to set the env variable just fine:

```
$> heroku run 'echo $DD_VERSION'
21ef0b58bacf156a4807b5dbc24494b7b29a5c4a
```

It's however not showing up in the APM / log tags. I'll share an update if I figure it out. :)

**Comment by pcreux** [comment]
:wave: So, `DD_VERSION` doesn't seem to be picked up by the agent. I reached out to Datadog support and they suggested to set the version in `prerun.sh` via `DD_TAGS`.

```sh
if [ -n "$HEROKU_SLUG_COMMIT" ]; then
  DD_TAGS=version:$HEROKU_SLUG_COMMIT
fi
```

The version shows up in the APM just fine.

**Comment by arapulido** [comment]
Thanks both! I agree with @pcreux that the prerun script is a better place to put this, to avoid making assumptions on what versions people want to track.

I will add the documentation for this and will investigate why `DD_VERSION` is not being picked up

**Comment by croaky** [comment]
Just deployed this change and it works great! Thanks again.

**Comment by pcreux** [comment]
Yes, https://github.com/DataDog/heroku-buildpack-datadog/releases/tag/1.19 has been released and has #201.
